index.html

```diff
@@ -915,12 +915,184 @@ <h4>Get a Specific Presentation</h4>
+      </p>
+      <ul class="bullet">
+        <li>
+`redirectUrl`: A URL that can be used to continue an interaction at another
+location. One use case for this is to send the user of an exchange client
+back to a coordinator website after an exchange has completed.
+        </li>
+        <li>
+`verifiablePresentation`: A Verifiable Presentation. This is used by either
+party in an exchange to provide information to the other party, either because
+the latter requested it or because the former is simply offering it.
+        </li>
+        <li>
+`verifiablePresentationRequest`: A Verifiable Presentation Request. This is
+used by either party in an exchange to request information from the other
+party.
+        </li>
+      </ul>
+      <p>
+Custom properties and values might also be included, but are expected to
+trigger errors in implementations that do not recognize them.
+      </p>
+      <p>
+To initiate an exchange using the VC API protocol, an exchange client
+performs an HTTP POST sending a JSON object as the request body. In the
+simplest case, when the client has no constraints of its own on the exchange
+&mdash; i.e., it has nothing to request from the other party &mdash; the
+JSON object is empty (`{}`). The workflow service responds with its own JSON
+object in the response body.
+      </p>
+      <p>
+If that response object is empty, the exchange is complete and nothing is
+requested from nor offered to the exchange client. If the object includes
+`verifiablePresentationRequest`, then the exchange is not yet complete and
+some additional information is requested, as specified by the contents of the
+associated verifiable presentation request. If the object includes
+`verifiablePresentation`, then some information is offered, such as
+verifiable credentials issued to the holder operating the exchange client or
+verifiable credentials with information about the exchange server's operator
+based on the exchange client's request. If the object includes `redirectUrl`,
+the exchange is complete and the workflow service recommends that the client
+proceed to another place to continue the interaction in another form.
+      </p>
+      <p>
+Many verifiable credential use cases can be implemented using these basic
+primitives. Either party to an exchange is capable of requesting verifiable
+presentations and of providing one or more verifiable credentials that might
+be necessary to establish trust and/or gain authorization capabilities, and
+either party is capable of presenting credentials that they hold or that they
+have issued. Specific workflows can be configured to expect specific
+presentations and credentials and to reject deviations from the expected flow
+of information. When a workflow service determines that a particular message
+is not acceptable, it raises an error by responding with a `4xx` HTTP status
+message and a JSON error object that expresses information about the error.
```

Comment:
```suggestion
message and a JSON object that expresses information about the error.
```

There is no "JSON error object" only objects which can contain information about errors. 😃